server/src/main/java/com/thoughtworks/go/config/GoConfigMigrator.java

```diff
@@ -88,6 +88,7 @@ public GoConfigHolder migrate() {
         try {
             return upgrade();
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Do we want to use the `LOGGER` as well?

Reply:
No logging is required here as the class contains a `UpgradeFailedHandler` (autowired constructor) which handles all the logging and thread exit.